docs/deployment/forked_threaded_servers.rst

```diff
@@ -0,0 +1,94 @@
+Forked and Threaded Servers
++++++++++++++++++++++++++++
+
+Forked and Threaded Servers share common "gotchas" and solutions when
+using pyramid and some popular packages.
+
+Forked and Threaded servers tend to use a "copy on write" implementation detail
+to optimize how they work and share memory. This can create problems when
+certain actions happen before the fork or thread dispatch, such as when files or
+file-descriptors are opened or random number generators are initialized.
+
+Many servers have built-in hooks or events which allow you to easily handle these
+situations.
+
+
+Servers
+++++++++++
+
+gunicorn
+========
+
+gunicorn offers several hooks during an application lifecycle
+
+the postfork routine is provided as a function in a configuration python script.
+
+for example a script 'config.py' might look like:
```

Comment:
For

For names of files and inline code, use double backticks:

```
``config.py``
```

Finally we want to use syntax highlighting, which we should explicitly state with a `.. code-block::` directive, and not rely on defaults.

```
For example a script ``config.py`` might look like the following.

.. code_block:: python
```